test/tests/restart/new_dt/new_dt_restart.i

```diff
@@ -0,0 +1,59 @@
+[Mesh]
+  file = new_dt_out_cp/0010_mesh.cpr
+  distribution = serial
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = CoefDiffusion
+    variable = u
+    coef = 0.1
+  [../]
+  [./time]
+    type = TimeDerivative
+    variable = u
+  [../]
+[]
+
+[BCs]
+  [./left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  [../]
+  [./right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  [../]
+[]
+
+[Executioner]
+  # Preconditioned JFNK (default)
+  type = Transient
+  num_steps = 10
+
+  # Here we are supplying a different dt
+  dt = 0.25
```

Comment:
Well depends on what you mean by work. Yes it picks up the new dt without the `reset_dt` flag. However if you look at the original ticket was this not the desired behavior?